modules/tensor_mechanics/src/materials/abaqus/AbaqusUMATStress.C

```diff
@@ -51,6 +51,8 @@ AbaqusUMATStress::validParams()
       "displaced mesh for computing displacements and quantities based on the deformed state.");
   params.addParam<UserObjectName>(
       "step_user_object", "The StepUserObject that provides times from simulation loading steps.");
+  params.addParam<RealVectorValue>("orientation",
+                                   "Euler angles corresponding to the Abaqus ORIENTATION keyword.");
```

Comment:
In my opinion we should not point to third party software documentation. Instead, describe.

Reply:
From the code, it looks it's a fixed orientation provided by the user.

Reply:
> In my opinion we should not point to third party software documentation. Instead, describe.

OK changed to "Euler angles that describe the orientation of the local material coordinate system."

> From the code, it looks it's a fixed orientation provided by the user.

It's an initial orientation (defined by some rotation away from the global) that is subsequently updated by the rotation increment at each time step.

Reply:
Sigh, explaining is fine, but a user coming from Abaqus who is looking for this functionality would have a much easier time if the doc string at least included which feature we are implementing here...

Reply:
Agree @dschwen. Perhaps the parameter name is a good enough hint

Reply:
I will yield to your judgement on this

Reply:
Yeah, maybe the name is enough of a hint. Either way, I don't think we should be beating around the bush and be upfront about what capability we are emulating here. Maybe mention "the Abaqus `ORIENTATION` keyword` in the .md file?

Reply:
>  Maybe mention "the Abaqus `ORIENTATION` keyword` in the .md file?

I can handle that